packages/poi-preset-elm/index.js

```diff
@@ -1,13 +1,16 @@
 const fs = require('fs')
+const path = require('path')
 
 module.exports = (options = {}) => {
   return poi => {
     const localCompiler = 'node_modules/.bin/elm-make'
     const loaderOptions = Object.assign({
       // Only use local compiler when it exists
-      pathToMake: fs.existsSync(localCompiler) ? localCompiler : null,
+      pathToMake: fs.existsSync(localCompiler) ? path.join(__dirname, localCompiler) : null,
```

Comment:
why `__dirname`?

Reply:
localCompiler value becomes relative to what `cwd` is. So if changed to something else the elm-make bin path is no longer valid. 

Should probably be `process.cwd()` as you suggest, but should not be left out.